src/main/java/appeng/me/storage/BasicCellInventoryHandler.java

```diff
@@ -43,9 +43,9 @@
  * @version rv6 - 2018-01-23
  * @since rv6 2018-01-23
  */
-public abstract class AbstractCellInventoryHandler<T extends IAEStack<T>> extends MEInventoryHandler<T> implements ICellInventoryHandler<T>
+public class BasicCellInventoryHandler<T extends IAEStack<T>> extends MEInventoryHandler<T> implements ICellInventoryHandler<T>
 {
-	public AbstractCellInventoryHandler( final IMEInventory c, final IStorageChannel<T> channel )
+	public BasicCellInventoryHandler( final IMEInventory c, final IStorageChannel<T> channel )
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') The Cyclomatic Complexity of this method "BasicCellInventoryHandler" is 14 which is greater than 10 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)